telegram/bot.py

```diff
@@ -1170,31 +1166,31 @@ def editMessageReplyMarkup(
         return url, data
 
     @log
-    def getUpdates(self, offset=None, limit=100, timeout=0, network_delay=.2):
+    def getUpdates(self, offset=None, limit=100, timeout=0, total_timeout=None):
```

Comment:
This will break for those who are using network_delay, but #310 adds **kwargs to all API calls, so I think we'll be good.